File: monitoring-proximity/models/content.js
var mongoose = require('mongoose');
var Schema = mongoose.Schema;

var ContentSchema = new Schema({
    uuid: {type:String, default: ''},
    minor: {type:Number, default: 0},
    major: {type:Number, default: 0},
    identifier: {type:String, default: ''},
    name: {type:String, default: ''},
    image_data: {type:String, default: ''}
});


var Content = mongoose.model('Content', ContentSchema);

module.exports = Content;

File: BeaconManagementServer/app/models/content.rb
class Content
  include Mongoid::Document
  field :uuid, type: String
  field :minor, type: Integer
  field :major, type: Integer
  field :identifier, type: String
  field :name, type: String
  field :image_data, type: String
  field :x, type: Integer
  field :y, type: Integer
end
